### ISSUE TITLE
Frozen rect while drawing keyboard

### ISSUE THREAD

**Issue by ppizarror** [Faylixe/pygame-vkeyboard#24]
Hi! I'm testing the library and found quite an interesting issue. While drawing the keyboard, a frozen rectangle above the keyboard is fixed, thus, is not updating the surface it should.

- Using pygame-menu v4.0.7
- Using pygame-vkeyboard v2.0.8
- Running on python 3.8 under windows 10 x64

My code:
```python
import pygame
import pygame_menu
from pygame_vkeyboard import *
from pygame_menu.examples import create_example_window

FPS = 30

surface = create_example_window('Example - Test', (500, 500))
clock = pygame.time.Clock()

menu = pygame_menu.Menu(
    height=300,
    theme=pygame_menu.themes.THEME_BLUE,
    title='Welcome',
    width=400
)

user_name = menu.add.text_input('Name: ', default='John Doe', maxchar=10)


def consumer(text: str) -> None:
    """
    Text from virtual keyboard to input.
    """
    user_name.set_value(text)


# Initializes and activates the keyboard
layout = VKeyboardLayout(VKeyboardLayout.AZERTY)
keyboard = VKeyboard(surface, consumer, layout)

while True:

    # Tick
    clock.tick(FPS)

    surface.fill((0, 0, 0))

    # Application events
    events = pygame.event.get()
    for event in events:
        if event.type == pygame.QUIT:
            exit()
        elif event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE:
            if keyboard.is_enabled():
                keyboard.disable()
            else:
                keyboard.enable()

    # Main menu
    menu.update(events)
    keyboard.update(events)

    menu.draw(surface)
    if keyboard.is_enabled():
        user_name.translate(0, -50)
        keyboard.draw(surface, force=True)
    else:
        user_name.translate(0, 0)

    # Flip surface
    pygame.display.flip()

```

Result:
![Animation2](https://user-images.githubusercontent.com/12925256/120076770-48cc7e80-c075-11eb-854e-e397c4792077.gif)

**Comment by anxuae** [comment]
Hi @ppizarror 

Nice to see you here ;-). I push a fix on the master branch which should fix the issue.

The keyboard use the DirtySprite mechanisms is order to avoid redrawing all the surface each time because it is time consuming, thus at first call of the `draw()`, a copy of the surface is kept and used to clear the "dirty" part of the screen.

In your example, the first call of `draw()` was done with the surface where the menu is not yet centered or correctly positioned, and it is kept in memory. When the "input box" of the keyboard have to be drawn (or cleared in your case because `show_text=False` in VKeyboard), the place where "input box" was foreseen is cleared with the copy in memory.

**Comment by anxuae** [comment]
Fixed in v2.0.9

**Comment by ppizarror** [comment]
Excellent, thanks!! 😄
